python/pyspark/ml/param/_shared_params_code_gen.py

```diff
@@ -115,7 +118,7 @@ def get$Name(self):
         ("outputCol", "output column name", None),
         ("numFeatures", "number of features", None),
         ("checkpointInterval", "checkpoint interval (>= 1)", None),
-        ("seed", "random seed", None),
+        ("seed", "random seed", "random.randint(0, sys.maxsize)"),
```

Comment:
Yes, in Scala, we use a random seed each time.  (See the code gen file for HasSeed, or several examples in spark.mllib.)  I think that's the best behavior: By default, we use a random seed.  If users want reproducible behavior, they should set the seed manually.